jcl/src/openj9.dtfj/share/classes/com/ibm/dtfj/javacore/parser/j9/section/thread/ThreadTagParser.java

```diff
@@ -95,18 +98,33 @@ private void initThreadInfoTag() {
 		ILineRule lineRule = new ThreadInfoLineRule();
 		addTag(IThreadTypes.T_3XMTHREADINFO, lineRule);
 	}
-	
-	
+
 	/**
-	 * 
 	 *
 	 */
+	private void initJavaLThrCCL() {
+		ILineRule lineRule = new ThreadInfoLineRule() {
+			public void processLine(String source, int startingOffset) {
```

Comment:
Please add the missing `@Override` annotation.

Reply:
Done